### ISSUE TITLE
[BUG]: XBSX2.0 won't read any dirs/files

### ISSUE THREAD

**Issue by widardd** [SirMangler/pcsx2#17]
### Describe the Bug

browsing files works, but no bios files or games are ever be found. after fiddling with this for an hour I found more people having this issue so I gave up.

installed that other port and everything worked instantly.

### Reproduction Steps

fire up dev mode, make sure the app is set to GAME

### PCSX2 Revision

XBSX2.0.2

### Operating System

Windows 11

**Comment by GABO1423** [comment]
Did you make sure that you setup the USB Drive properly and that the Xbox properly recognizes it?

**Comment by widardd** [comment]
> Did you make sure that you setup the USB Drive properly and that the Xbox properly recognizes it?

Yes, RetroArch, PPSSPP and AetherSX2 work. The latter finds my bios and games and runs everything. XBSX2 doesn't recognize anything.

**Comment by Pierocubo** [comment]
Same issue here, there is any solution? Thanks in advance!

**Comment by SirSamael** [comment]
I'm also having the same issue with this new version..do anyone have any solution yet?

**Comment by Diego90-sn** [comment]
Eu também estou tendo esse mesmo problema.
Alguém consegui resolver esse problema ?
